CONTRIBUTING.md

```diff
@@ -0,0 +1,338 @@
+# Apache APISIX Blog Contributing Guide
```

Comment:
How about putting this guide under the Community section? We have a related page here: 

https://apisix.apache.org/docs/general/community/

😄 If we put this file under the repo directly, users have to read docs from markdown file.

Reply:
The comment above is optional, contents and workflow in this guide is more important.